dtests/src/test/java/io/snappydata/hydra/security/SnappySecurityTest.java

```diff
@@ -0,0 +1,464 @@
+                execute(queryStr, conn);
+              }
+            }
+          }
+        } else {
+          Log.getLogWriter().info("The query to be executed is  " + queryStr + " with new authorization = " + isAuth);
+          execute(queryStr, conn);
+        }
+      } catch (SQLException e) {
+        if (isAuth) {
+          unExpectedExceptionCnt = unExpectedExceptionCnt + 1;
+          Log.getLogWriter().info(" unExpectedExceptionCnt Count is " + unExpectedExceptionCnt);
+          Log.getLogWriter().info("Got UnExpected Exception " + e.getMessage());
+        } else {
+          expectedExceptionCnt = expectedExceptionCnt + 1;
+          Log.getLogWriter().info("Got Expected Exception " + e.getMessage());
+        }
+      }
+    }
+    closeConnection(conn);
+  }
+
+  public static void execute(String queryStr, Connection conn) throws SQLException {
+    if (queryStr.contains("SELECT"))
+      conn.createStatement().executeQuery(queryStr);
+    else
+      conn.createStatement().execute(queryStr);
+    Log.getLogWriter().info("Query executed successfully");
+  }
+
+  public static void validate(Integer expectedCnt, Integer unExpectedCnt) {
+    if (unExpectedExceptionCnt != unExpectedCnt)
+      throw new TestException("The Result is WRONG :Expected unExpectedExceptionCnt = " + unExpectedCnt + " but got " +
+          unExpectedExceptionCnt);
+    else
+      Log.getLogWriter().info("Successfully Got expected unExpectedExceptionCnt " + unExpectedExceptionCnt);
+
+    if (expectedExceptionCnt != expectedCnt)
+      throw new TestException("The Result is WRONG :Expected expectedExceptionCnt = " + expectedCnt + " but got " +
+          expectedExceptionCnt);
+    else
+      Log.getLogWriter().info("Successfully Got expected expectedExceptionCnt " + expectedExceptionCnt);
+
+    unExpectedExceptionCnt = 0;
+    expectedExceptionCnt = 0;
+  }
+
+  public static Connection getSecuredLocatorConnection(String usr, String pass) throws SQLException {
+    List<String> endpoints = validateLocatorEndpointData();
+    Properties props = new Properties();
+    props.setProperty("user", usr);
+    props.setProperty("password", pass);
+    Connection conn = null;
+    String url = "jdbc:snappydata://" + endpoints.get(0) + "/";
+    conn = getConnection(url, "io.snappydata.jdbc.ClientDriver", props);
+    return conn;
+  }
+
+  private static Connection getConnection(String protocol, String driver, Properties props)
+      throws
+      SQLException {
+    Log.getLogWriter().info("Creating secure connection using " + driver + " with " + protocol +
+        " and credentials = " + props.getProperty("user") + props.getProperty("password"));
+    loadDriver(driver);
+    Connection conn = DriverManager.getConnection(protocol, props);
+    return conn;
+  }
+
+  public static synchronized void HydraTask_executeSQLScripts() {
```

Comment:
This method seems unnecessary as except security parameters handling rest all is same as SnappyTest.executeSQLScript.

Add the security parameters handling in original method itself.
This can be done later after 1.0 as well.